rust/operator-binary/src/controller.rs

```diff
@@ -1364,16 +1364,25 @@ async fn get_proxy_hosts(
     nifi: &NifiCluster,
     nifi_service: &Service,
 ) -> Result<String> {
+    let host_header_check = nifi.spec.cluster_config.host_header_check.clone();
+
+    if host_header_check.allow_all {
+        tracing::info!("spec.clusterConfig.hostHeaderCheck.allowAll is set to true. All proxy hosts will be allowed.");
```

Comment:
We could also trace that `additionalAllowedHosts` will be ignored.

Reply:
I've added an additional trace only when `additionalAllowedHosts` is not empty and will be ignored. My assumption is that this is then probably not behavior intended by the user.